src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,250 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+    public static final int MAX_TASKS = 100;
+    private static int taskCount = 0;
+    static Task[] tasks = new Task[MAX_TASKS];
+
+    static String[] commands = {"todo", "mark", "unmark", "deadline", "list", "help", "event"};
+
+    public static void addTask(Task t) {
```

Comment:
Try to give variables more descriptive names like `Task task` so that it would not be confusing to readers when your methods are longer